src/nvim/charset.c

```diff
@@ -43,20 +43,29 @@ static bool chartab_initialized = false;
 #define GET_CHARTAB(buf, c) \
     ((buf)->b_chartab[(unsigned)(c) >> 6] & (1ull << ((c) & 0x3f)))
 
-/// Fill chartab[].  Also fills curbuf->b_chartab[] with flags for keyword
+// Table used below, see init_chartab() for an explanation
+static char_u g_chartab[256];
+
+// Flags for g_chartab[].
+#define CT_CELL_MASK  0x07  // mask: nr of display cells (1, 2 or 4)
+#define CT_PRINT_CHAR 0x10  // flag: set for printable chars
+#define CT_ID_CHAR    0x20  // flag: set for ID chars
+#define CT_FNAME_CHAR 0x40  // flag: set for file name chars
```

Comment:
Please change the comments here to `///<`, so it will be picked up by [Doxygen](https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html).